next.json

```diff
@@ -762,6 +762,13 @@
                     {"name": "output attachment location", "type": "uint32_t"},
                     {"name": "attachment slot", "type": "uint32_t"}
                 ]
+            },
+            {
+                "name": "subpass set depth stencil attachment",
```

Comment:
Oh yeah, I intended to split this. From the APIs I've looked at, you just set the two separately, and they can point at the same TextureView if it's a combined format.